src/main/java/com/dev/cinema/service/OrderServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.dev.cinema.service;
+
+import com.dev.cinema.dao.OrderDao;
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.User;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
```

Comment:
IMHO it will be better to use shoppingCartDao here and not to use one service inside another one

Reply:
we don't have `clear()` in shoppingCartDao, what you suggest leads to code duplication